src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,23 @@
+import { Person } from '../../types/Person';
+import './PersonInfo.scss';
+
+type Props = {
+  person: Person,
+  onClick: (person: Person) => void,
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, onClick }) => {
+  return (
+    <div
+      className="dropdown-item"
+      onClick={() => onClick(person)}
+      onKeyUp={() => {}}
```

Comment:
The onKeyUp event handler is currently an empty function, which doesn't provide any functionality. Consider implementing a proper event handling for keyboard events or remove this line if it's not necessary.